cmake/Modules/FindMKLDNN.cmake

```diff
@@ -42,16 +42,17 @@ IF(NOT MKLDNN_FOUND)
         list(APPEND DNNL_MAKE_COMMAND "--" "-l" "$ENV{MAX_JOBS}")
       endif()
     endif()
+    set(DNNL_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
```

Comment:
I wonder if `CMAKE_CXX_FLAGS` here still is the default one. In PyTorch cmake system, there are a lot of adds in `CMAKE_CXX_FLAGS`. If `CMAKE_CXX_FLAGS` is not the default one here, is it risky involve PyTorch added flags in oneDNN build?

Reply:
@fengyuan14 , thanks. It's a valid point. Updated.